src/core/transactions/ViewTransferTransaction.ts

```diff
@@ -25,9 +25,23 @@ import { MosaicModel } from '@/core/database/entities/MosaicModel';
 export class ViewTransferTransaction extends TransactionView<TransferTransaction> {
     public get isIncoming() {
         const currentSignerAddress = this.$store.getters['account/currentSignerAddress'];
+        if (this.transaction.type === TransactionType.TRANSFER && this.transaction.recipientAddress instanceof NamespaceId) {
+            const linkedAddress = this.$store.getters['namespace/linkedAddress'];
+            if (!linkedAddress) {
+                this.checkLinkedAddress(this.transaction.recipientAddress);
+                return false;
```

Comment:
why return false here, does it needs to recheck the linkedAddress after dispatch called?

Reply:
yes, as GET_LINKED_ADDRESS is an async function and doesn't return the linked address so there must be a return statement to avoid sending multiple requests to fetch namespace linkedAdress.